source/client/service_impl.cc

```diff
@@ -0,0 +1,103 @@
+#include "client/service_impl.h"
+
+#include <grpc++/grpc++.h>
+
+#include "client/client.h"
+#include "client/options_impl.h"
+
+namespace Nighthawk {
+namespace Client {
+
+void ServiceImpl::handleExecutionRequest(const nighthawk::client::ExecutionRequest& request) {
+  RELEASE_ASSERT(running_, "running_ ought to be set");
```

Comment:
Is this an invariant? Should it just be `ASSERT`?

Reply:
Fixed